packages/datetime/src/dateRangeInput.tsx

```diff
@@ -512,6 +753,10 @@ export class DateRangeInput extends AbstractComponent<IDateRangeInputProps, IDat
         return (boundary === DateRangeBoundary.START) ? DateRangeBoundary.END : DateRangeBoundary.START;
     }
 
+    private areSameDay = (a: moment.Moment, b: moment.Moment) => {
```

Comment:
Would it need to be `areSameMomentDay` though? Same "moment" implies millisecond precision to me.